src/Simplex/Chat/Controller.hs

```diff
@@ -420,7 +446,16 @@ data ChatCommand
   deriving (Show)
 
 data ChatResponse
-  = CRActiveUser {user :: User}
+  = CRZoneCreated {ident :: ZoneId}
+  | CRZoneList {zones :: [ZoneId]}
+  | CRZoneDetails {ident :: ZoneId, title :: Text, active :: Bool, kind :: ZoneKind}
```

Comment:
don't think you need to different responses - can just return list of objects, no need to get them one by one - just harder to use API. You need a type for this record.